### ISSUE TITLE
onlineshowanswers does not do anything in OSU css

### ISSUE THREAD

**Issue by clairemerriman** [XimeraProject/ximeraExperimental#7]
https://github.com/XimeraProject/ximeraExamples/blob/6de296851da03c4c738200f83f0fc2697886ffd6/answers/answerBox.tex#L77

**Comment by wiobber** [comment]
Indeed, this needs changes to the ximeraServer code ... Changes of KULeuven are still to be integrated @OSU...

**Comment by clairemerriman** [comment]
Mostly a comment to myself to consider the wording of the sentence.

**Comment by wiobber** [comment]
Closed ...
